R/mobr_boxplots.R

```diff
@@ -289,9 +395,12 @@ plot.mob_stats = function(mob_stats,
                   main = panel_title, ...)
           if (multipanel)
```

Comment:
This is a lot of code replication in this portion of the function. If possible in the future it would be great to generalize the code by looping through the metric names when building the boxplots. Just a suggestion for later on.